integration-tests/alias-test/generate/src/test/python/aliastest/generate/mbean_info_helper.py

```diff
@@ -164,18 +174,21 @@ def __get_mbean_interface(self):
             interfaces = self.__mbean_instance.getClass().getInterfaces()
             if len(interfaces) > 0:
                 if len(interfaces) > 1:
-                    self.__logger.fine('WLSDPLYST-01337', str(interfaces), self.__mbean_instance,
+                    self.__logger.fine('There are multiple Mbean interface{0} for mbean instance {1}',
```

Comment:
Should the text `There are multiple Mbean interface{0} for mbean instance {1}`  be in the message bundle?

Reply:
There is no message bundle for alias test any longer...